RxCocoa/Common/RxTextInput.swift

```diff
@@ -21,6 +21,19 @@ import Foundation
         */
         var rx_text: ControlProperty<String> { get }
     }
+
+    extension UITextField : RxTextInput {
+        public var rx_text: ControlProperty<String> {
+            return self.rx.text
+        }
+    }
+
+    extension UITextView : RxTextInput {
+        public var rx_text: ControlProperty<String> {
+            return self.rx.text
+        }
+    }
```

Comment:
It would be great to solve this somehow.

I wish something like this would work:
```swift
/**
    Represents text input with reactive extensions.
    */
    public protocol RxTextInput : UITextInput {
        /**
         Reactive wrapper for `text` property.
        */
        static func text(extension: Self) -> ControlProperty<String>
    }

    extension UITextField : RxTextInput {
        public static func text(extension: Self) -> ControlProperty<String> {
            return `extension`.rx.text
        }
    }

    extension UITextView : RxTextInput {
        public static func text(extension: Self) -> ControlProperty<String> {
            return `extension`.rx.text
        }
    }
```

... but it doesn't :(